clients/apps/web/src/hooks/queries/subscriptions.ts

```diff
@@ -1,15 +1,23 @@
-import { SubscriptionsApiListRequest } from '@polar-sh/sdk'
-import { useQuery } from '@tanstack/react-query'
+import { SubscriptionsApiListRequest, SubscriptionCancel } from '@polar-sh/sdk'
+import { useQuery, useMutation } from '@tanstack/react-query'
 
-import { api } from '@/utils/api'
+import { api, queryClient } from '@/utils/api'
 import { defaultRetry } from './retry'
 
+export const invalidateSubscriptionCache = (
+  organizationId: string,
+) => {
+  queryClient.invalidateQueries({
+    queryKey: ['subscriptions', organizationId],
+  })
+}
+
 export const useListSubscriptions = (
   organizationId?: string,
   parameters?: Omit<SubscriptionsApiListRequest, 'organization_id'>,
 ) =>
   useQuery({
-    queryKey: ['subscriptions', { organizationId, ...(parameters || {}) }],
+    queryKey: ['subscriptions', organizationId, { ...(parameters || {}) }],
```

Comment:
I would prefer to keep everything as an object, much more [versatile](https://tanstack.com/query/latest/docs/framework/react/guides/query-invalidation).